changelog/62848.fixed

```diff
@@ -0,0 +1 @@
+Clarify pkg.installed's pkg_verify option documentation
```

Comment:
I think this is the wrong changelog file and description for this change

Reply:
Whoops, how did that sneak in? :facepalm: I'll fix this today.